test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -291,8 +291,9 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 	})
 
 	Context("RPS configuration", func() {
-		It("Should have the correct RPS configuration", func() {
-			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved != nil {
+		It("[test_id:54190] Should have the correct RPS configuration", func() {
+			if profile.Spec.CPU == nil || profile.Spec.CPU.Reserved == nil {
+				testlog.Warning("Test Skipped due nil Reserved cpus")
```

Comment:
since we want to fix it for good, let's just use a `ginkgo.Skip` here instead of reinventing it partially.